magprime/automated_emails.py

```diff
@@ -34,28 +34,28 @@ def __init__(self, event):
                sender='MAGFest LAN <lan@magfest.org>')
 
 AutomatedEmail(Attendee, 'MAGFest Tech Ops volunteering', 'techops.txt',
-           lambda a: a.staffing and a.requested(c.TECH_OPS) and not a.assigned_to(c.TECH_OPS), sender='MAGFest TechOps <techops-heads@magfest.org>')
+           lambda a: a.staffing and a.assigned_to(c.TECH_OPS), sender='MAGFest TechOps <techops-heads@magfest.org>')
 
 AutomatedEmail(Attendee, 'MAGFest Chipspace volunteering', 'chipspace.txt',
-           lambda a: a.staffing and (a.requested(c.JAMSPACE) or a.assigned_to(c.JAMSPACE)) and not a.assigned_to(c.CHIPSPACE), sender='MAGFest ChipSpace <chipspace@magfest.org>')
+           lambda a: a.staffing and a.assigned_to(c.CHIPSPACE), sender='MAGFest ChipSpace <chipspace@magfest.org>')
```

Comment:
Does this email still need to exist?  The reason why it was originally configured was because we didn't have Chipspace listed as a department people could express interest in, so we emailed all of the people who said they were interested in jamspace, reasoning that the two are similar enough.